### DISCUSSION TITLE
msvc编译不通过

### DISCUSSION THREAD

**Discussion by mrZhang13** [ejoy/ant#24] [Q&A] [answered]
编译ltask的时候，提示无法打开 'stdatomic.h'

**Comment by zeromake** [comment]
@mrZhang13 可以去 [dav1d](https://github.com/videolan/dav1d/blob/master/include/compat/msvc) 摸一个

**Comment by actboy168** [comment, marked as answer]
stdatomic.h需要msvc 17.5或以上

**Comment by oahzuw** [comment]
Visual Studio Community 2022 17.8.4 编译通过，test 目录下demo都可以跑
